### PR TITLE
Add xpnet start/stop

### PR DESCRIPTION
**Pull request by NStetskovych**
Signed-off-by: Nadiya.Stetskovych <Nadiya.Stetskovych@cavium.com>

### CODE REVIEW

File: ansible/roles/sonicv2/templates/etc/systemd/system/syncd.j2 [hunk 1]
```diff
@@ -17,6 +20,9 @@ ExecStop=/usr/bin/docker stop syncd
 ExecStopPost=/etc/mlnx/msn2700 stop
 ExecStopPost=/etc/init.d/sxdkernel stop
 ExecStopPost=/usr/bin/mst stop
+{% elif sonic_hwsku == 'AS7512' %}
+ExecStopPost=/etc/init.d/xpnet.sh stop
+ExecStopPost=/etc/init.d/xpnet.sh start
```
Comment (stcheng) [inline ansible/roles/sonicv2/templates/etc/systemd/system/syncd.j2]: why xpnet.sh will be started after the docker syncd stops?

Comment (stcheng) [inline ansible/roles/sonicv2/templates/etc/systemd/system/syncd.j2]: @NStetskovych

Comment (NStetskovych) [inline ansible/roles/sonicv2/templates/etc/systemd/system/syncd.j2]: @stcheng for example for ptf testing (when syncd stopped and saiserver deployed)

Comment (stcheng) [inline ansible/roles/sonicv2/templates/etc/systemd/system/syncd.j2]: I see. then i'm just wondering if this logic is better embedded to the saiserver or somewhere else. thanks for answering my question.

**Pull request merged**